Description: hardcode configdir /etc/moin for "moin" script
 only applicable on systems like Debian using a system-wide config area
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2009-11-15
--- a/setup.py
+++ b/setup.py
@@ -170,9 +170,8 @@
                         % script_vars)
                 else:
                     file.write("#! %(python)s\n"
-                        "#Fix and uncomment those 2 lines if your moin command doesn't find the MoinMoin package:\n"
-                        "#import sys\n"
-                        "#sys.path.insert(0, '%(package_location)s')\n"
+                        "import sys\n"
+                        "sys.path.insert(0, '/etc/moin')\n"
                         "from %(package)s.script.%(module)s import run\n"
                         "run()\n"
                         % script_vars)
